less/PaymentAccordion.less

```diff
@@ -0,0 +1,42 @@
+.mitt-konto--payment-header {
```

Comment:
Right. The convention is also to name the classes `.<component-name>--class-name` so for instance `.payment-accordion--payment-header` here 🙂 It makes inspecting the DOM so much nicer.

Reply:
Ok, I updated the class names. Though I loaned `mitt-konto--component-block-content` for my styling and I didn't copy that under a different name.